tests/tests/fv3_ccpp_hwrf_rrtmg

```diff
@@ -0,0 +1,85 @@
+###############################################################################
+#
+#  FV3 CCPP GFS v15.2 compiled with 32-bit dynamics test
+#
+###############################################################################
+
+export TEST_DESCR="Compare FV3 32bit CCPP GFS HWRF RRTMG results with previous trunk version"
```

Comment:
Is the test really compiled with 32-bit dynamics?

Reply:
no. I copied a old template.